README.md

```diff
@@ -42,30 +49,21 @@ Rector **instantly upgrades and instantly refactors the PHP code of your applica
     <img src="/docs/images/space.png" width=40>
     <a href="/config/set/twig"><img src="/docs/images/twig.png"></a>
 </p>
-
-<br>
-
-## What Can Rector Do for You?
-
-- Rename classes, methods, properties, namespaces or constants
-- Complete [parameter, var or return type declarations](https://www.tomasvotruba.cz/blog/2019/01/03/how-to-complete-type-declarations-without-docblocks-with-rector/) based on static analysis of your code
-- Upgrade your code from PHP 5.3 to PHP 7.4
-- [Migrate your project from Nette to Symfony](https://www.tomasvotruba.cz/blog/2019/02/21/how-we-migrated-from-nette-to-symfony-in-3-weeks-part-1/)
-- [Complete PHP 7.4 property type declarations](https://www.tomasvotruba.cz/blog/2018/11/15/how-to-get-php-74-typed-properties-to-your-code-in-few-seconds/)
-- [Refactor Laravel facades to dependency injection](https://www.tomasvotruba.cz/blog/2019/03/04/how-to-turn-laravel-from-static-to-dependency-injection-in-one-day/)
-- And much more...
-
 ...**look at the overview of [all available Rectors](/docs/AllRectorsOverview.md)** with before/after diffs and configuration examples. You can use them to build your own sets.
+<br>
 
-## How to Apply Coding Standards?
+## Integrating Coding Standards?
```

Comment:
Not sure what does that mean now, Rector doesn't integrate coding standard, nor you can integrate coding standards into it

Reply:
Can you run a coding standards via Rector? If yes, then it's integrated and if no, i'm mistaken.